atomic_reactor/plugins/pre_bump_release.py

```diff
@@ -137,6 +138,9 @@ def run(self):
 
         if self.append:
             next_release = self.get_next_release_append(component, version, release)
+        elif is_scratch_build():
```

Comment:
Shouldn't be scratch build first case to test to override other build options?

Reply:
that append will work even when "wrong" release is specified as it is just appending additional string to already existing release, so we can keep that functionality

Reply:
From my POW, scratch builds are isolated from any releases, so I assume that once --scratch is used, we shouldn't even touch koji for existing releases.

This makes results inconsistent.